app/views/components/TaxReturnLink.scala.html

```diff
@@ -26,5 +26,5 @@
 
 <li id=@returnItem class="govuk-list">
  <p class="govuk-heading-m">@message
- <br><a class="govuk-body" href="@call">@linkText</a></p>
+ <br><a class="govuk-body" aria-label="@message" href="@call">@linkText</a></p>
```

Comment:
i assume this message has arleady been put through the messages file, and this is the result